scss/layout/_layout.scss

```diff
@@ -139,6 +139,18 @@
   padding-right: u(2rem);
 }
 
+.u-small-icon-padding--left {
+  padding-left: u(2.5rem);
+}
+
+// horizontal rule
+
+.hr--light {
```

Comment:
There's a similar class in `_contact-form.scss` for `.hr--light`. Wanna get rid of that one and also move `.hr--dark` here so they can be together?